cookbooks/aws-parallelcluster-slurm/templates/default/slurm/slurm.conf.erb

```diff
@@ -37,6 +37,12 @@ ResumeTimeout=1800
 PrivateData=cloud
 ResumeRate=0
 SuspendRate=0
+<% if node["cluster"]["workaround_with_private_hostname"] == 'true' -%>
+Epilog=/opt/slurm/etc/pcluster/epilog
+Prolog=/opt/slurm/etc/pcluster/prolog
+PrologFlags=alloc,serial
+SchedulerParameters=nohold_on_prolog_fail
+<% end -%>
```

Comment:
Can we add a comment on top of these block of options to briefly explain what they are for?

Reply:
Done